setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal = 1
 [flake8]
 application-import-names = orderable
-ignore = D100,D101,D102,D203,D204
+ignore = D100,D101,D102,D104,D105,D203,D204
```

Comment:
good old `flake8-docstrings`.  What do you (and the rest of @incuna/backend) think about just not using it?  We're ignoring quite a lot of the warnings here - is what's left even all that useful?